dlls/ff/ff_lualib_team.cpp

```diff
@@ -32,6 +33,29 @@ std::ostream& operator<<(std::ostream& stream, const CTeam& team)
 	return stream << "team" << ":" << const_cast<CTeam&>(team).GetName() << ":" << const_cast<CTeam&>(team).GetTeamNumber();
 }
 
+namespace FFLib
+{
+	// helper function to turn the bitmask into a lua table of teams
+	luabind::adl::object GetAllies(CFFTeam *pTeam)
+	{
+		luabind::adl::object luatblAllies = luabind::newtable(_scriptman.GetLuaState());
+		if (pTeam)
+		{
+			int iTableKey = 1;
+			int alliesMask = pTeam->GetAllies();
+			for (int teamId = TEAM_UNASSIGNED; teamId < TEAM_COUNT; teamId++)
+			{
+				if (alliesMask & (1<<teamId))

```

Comment:
You can reduce nesting here by inverting the if and using continue (i prefer early-exit over nested if statements)